testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/resource/param/CookieInjectionTest.java

```diff
@@ -112,6 +124,38 @@ public void testCookieExpire() {
             throw new RuntimeException(e);
         }
     }
+
+    /**
+     * @tpTestDetails Cookie in the response header contains correct "Expires" attribute. Tested for cookie version 0 and 1.
+     * See RESTEASY-1476 for details.
+     * @tpSince RESTEasy 3.1.0
+     */
+    @Test
+    @Category({NotForForwardCompatibility.class})
+    public void testCookieExpire() {
+        _testExpire("/expire");
+        _testExpire("/expire1");
+        _testExpire("/expired");
+    }
+
+    /**
+     * @tpTestDetails Cookie in the response header contains correct "Expires" attribute. Tested for cookie version 0 and 1.
+     * Proxy is used. See RESTEASY-1476 for details.
+     * @tpSince RESTEasy 3.1.0
+     */
+    @Test
+    @Category({NotForForwardCompatibility.class})
+    public void testProxyExpire() {
+        CookieProxy proxy = ProxyBuilder.builder(CookieProxy.class, client.target(generateURL("/"))).build();
+        Response response = proxy.expire();
+        String res = response.readEntity(String.class);
+        System.out.println(res);
+        MultivaluedMap<String, String> headers = response.getStringHeaders();
+        System.out.println(headers.get("Set-Cookie"));

```

Comment:
Right, it doesn't, forgot to remove it. It is removed now. Thank you for review.